pandas/tests/config/test_config.py

```diff
@@ -14,9 +14,9 @@ def setup_class(cls):
         from copy import deepcopy
 
         cls.cf = cf
-        cls.gc = deepcopy(getattr(cls.cf, "_global_config"))
-        cls.do = deepcopy(getattr(cls.cf, "_deprecated_options"))
-        cls.ro = deepcopy(getattr(cls.cf, "_registered_options"))
+        cls._global_config = deepcopy(getattr(cls.cf, "_global_config"))
+        cls._deprecated_options = deepcopy(getattr(cls.cf, "_deprecated_options"))
+        cls._registered_options = deepcopy(getattr(cls.cf, "_registered_options"))
```

Comment:
Needed to change cls.ro and took the liberty to do the same with the others to keep a standard